lang/python/pytest/Makefile

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (C) 2019 CZ.NIC, z. s. p. o. (https://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=pytest
+PKG_VERSION:=5.0.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=pytest-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://files.pythonhosted.org/packages/source/p/pytest/
+PKG_HASH:=95b700cf21ed5b7e91bce7a6b5a573b2e3ef7b3643d00f681d8f9c4672f9fbdf
+
+PKG_MAINTAINER:=Jan Pavlinec <jan.pavlinec@nic.cz>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+PYTHON3_PKG_SETUP_VARS:= \
+        PKG_VERSION="$(PKG_VERSION)"
+
+define Package/python3-pytest
+  SUBMENU:=Python
+  SECTION:=lang
+  CATEGORY:=Languages
+  TITLE:=Python testing framework
+  URL:=https://docs.pytest.org/en/latest/
+  DEPENDS:= \
+	+python3-light \
+	+python3-setuptools \
+	+python3-atomicwrites \
+	+python3-more-itertools \
+	+python3-py \
+	+python3-six \
```

Comment:
[six is no longer a dependency](https://github.com/pytest-dev/pytest/blob/5.0.0/setup.py#L5-L16).